Makefile.in

```diff
@@ -186,17 +186,43 @@ tags: $(TAGFILES)
 TAGS: $(TAGFILES)
 	etags $(TAGFILES)
 
+#
+# Use git archive piped to tar to construct a subdirectory whose name
+# is libpcap-{release}, containing all the checked-in source files,
+# and then run autoreconf in that directory to generate the configure
+# script and other files from that source.  Then remove autom4te.cache,
+# construct the release tarball from that subdirectory, and remove
+# the subdirectory.
+#
+# The ^{tree} is there to force git archive not to write out the
+# "helpful" global extended pax header with a commit ID, as not all
+# versions of tar can handle that (Solaris tar can't, for example).
+# (It turns HEAD, or a tag, both of which are apparently "tree-ish"es,
+# into a tree; apparently, unlike HEAD, or a tag, which have a commit
+# ID associated with them, the tree associated with them doesn't have
+# a commit ID, so no commit ID is available to write, and thus
+# git archive doesn't write one.)
+#
 releasetar:
 	@TAG=$(PROG)-`cat VERSION` && \
-	if git show-ref --tags --quiet --verify -- "refs/tags/$$TAG"; then \
-	    git archive --prefix="$$TAG"/ -o "$$TAG".tar.gz "$$TAG" \
-	    $(RELEASE_FILES) && \
+	AUTORECONF_DIR=/tmp/autoreconf_"$$TAG"_$$$$ && \
+	DIR=`pwd` && \
+	cd /tmp && \
+	rm -rf "$$AUTORECONF_DIR" && \
+	mkdir "$$AUTORECONF_DIR" && \
+	cd "$$AUTORECONF_DIR" && \
+	if git -C $$DIR show-ref --tags --quiet --verify -- "refs/tags/$$TAG"; then \
+	    (git -C $$DIR archive --prefix="$$TAG"/ "$$TAG"^{tree} $(RELEASE_FILES) | \
+	     tar xf -) && \
 	    echo "Archive build from tag $$TAG."; \
 	else \
-	    git archive --prefix="$$TAG"/ -o "$$TAG".tar.gz HEAD \
-	    $(RELEASE_FILES) && \
+	    (git -C $$DIR archive --prefix="$$TAG"/ HEAD^{tree} $(RELEASE_FILES) | tar xf -) && \
 	    echo "No $$TAG tag. Archive build from HEAD."; \
-	fi
+	fi && \
+	(cd "$$TAG" && autoreconf && rm -rf autom4te.cache) && \
+	tar cf "$$DIR/$$TAG".tar.gz "$$TAG" && \
```

Comment:
`cf` -> `czf` ?

Reply:
Or use gzip ?

Reply:
Previous comment "AIX tar does not support "z", so use a pipe for gzip."

Reply:
Done.